src/ucp/api/ucp.h

```diff
@@ -409,7 +409,7 @@ enum ucp_worker_attr_field {
     UCP_WORKER_ATTR_FIELD_MAX_AM_HEADER = UCS_BIT(3), /**< Maximal header size
                                                            used by UCP AM API */
     UCP_WORKER_ATTR_FIELD_NAME          = UCS_BIT(4), /**< UCP worker name */
-    UCP_WORKER_ATTR_FIELD_MAX_DEBUG_STR = UCS_BIT(5)  /**< Maximum debug string */
+    UCP_WORKER_ATTR_FIELD_MAX_INFO_STR  = UCS_BIT(5)  /**< Maximum debug string */
```

Comment:
sorry, just noticed : better UCP_WORKER_ATTR_FIELD_MAX_INFO_STR**ING** to match other macros